File: level_1/나누어_떨어지는_숫자_배열/devSOWON0628.js
function solution(arr, divisor) {
    let answer = [];
    for(let i = 0 ; i < arr.length ; i++){
        if(arr[i] % divisor == 0){
            answer.push(arr[i])
        }
    }
    if(answer.length == 0){
        return [-1]
    }else{
        answer = answer.sort((a,b)=>{
            if(a>b){
                return 1
            }else if(a<b){
                return -1
            }else{
                return 1
            }
        })
    }
    
    return answer;
}

File: level_2/전화번호_목록/devSOWON0628.java
import java.util.*;
class Solution {
    public boolean solution(String[] phone_book) {
        String[] sortedPhoneBook = phone_book.clone();
        Arrays.sort(sortedPhoneBook);
            
        for (int i = 0; i < sortedPhoneBook.length - 1; i++) {
            if (sortedPhoneBook[i + 1].startsWith(sortedPhoneBook[i])) {
                return false;
            }
        }
        return true;
    }
}
